packages/containers.nix

```diff
@@ -108,13 +108,14 @@ let
       name = "service-mesh-proxy";
       tag = "v${pkgs.service-mesh.version}";
       copyToRoot = with pkgs; [
+        busybox
         envoy
         iptables-legacy
       ];
       config = {
         # Use Entrypoint so we can append arguments.
         Entrypoint = [ "${pkgs.service-mesh}/bin/service-mesh" ];
-        Env = [ "PATH=/bin" ]; # This is only here for policy generation.
+        Env = [ "PATH=/bin" ];
```

Comment:
nit, as it was here before already. But one could theoretically use [`buildEnv`](https://noogle.dev/f/pkgs/buildEnv) here.

Reply:
How would `buildEnv` help here?

Reply:
It would automagically give you an image with the correct PATH. See [this example](https://github.com/NixOS/nixpkgs/blob/fa39fa603cde60b27fa1a8b01c24b0be45362a6f/pkgs/build-support/docker/examples.nix#L86-L94)

Reply:
Afaiu `buildEnv` can help set symlinks, but does it also automagically set the `Env` image configuration? We need that, otherwise policy generation fails.